AdobeStockImage/Model/GetRelatedImages.php

```diff
@@ -17,10 +17,20 @@
 use Psr\Log\LoggerInterface;
 
 /**
- * Class GetImageSeries
+ * Class GetRelatedImages
  */
-class GetImageSeries
+class GetRelatedImages
 {
+    /*
+     * Series ID.
+     */
+    const SERIE_ID = 'serie_id';
```

Comment:
To avoid code duplication, it's better to use the field as a method parameter or as a constructor parameter provided by DI and use cycles instead of similar code blocks

Reply:
Hello @sivaschenko ,

We can avoid code duplication by replacing `Magento\Framework\Api\Search\SearchCriteriaBuilder` to `Magento\Framework\Api\SearchCriteriaBuilder` so that we can pass array of filter but we have already used `Magento\Framework\Api\Search\SearchCriteriaBuilder` in so many places that there are lot of chance some functionality might broke. Let me know if you want to move forward with this logic or any other logic from your side is always welcome

Thanks!